sdk/eventhubs/azure-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/ClientConstants.java

```diff
@@ -8,13 +8,12 @@
 public final class ClientConstants {
     public static final Duration OPERATION_TIMEOUT = Duration.ofSeconds(60);
     public static final String NOT_APPLICABLE = "n/a";
-    public static final int HTTPS_PORT = 443;
     public static final int MAX_EVENTHUB_AMQP_HEADER_SIZE_BYTES = 512;
     public static final Duration TOKEN_VALIDITY = Duration.ofMinutes(20);
     public static final int SERVER_BUSY_BASE_SLEEP_TIME_IN_SECS = 4;
 
     public static final String PRODUCT_NAME = "azsdk-java-eventhubs";
-    public static final String CURRENT_JAVACLIENT_VERSION = "1.0.0-SNAPSHOT";
+    public static final String CURRENT_JAVA_CLIENT_VERSION = "1.0.0-preview.2";
```

Comment:
You should file an issue for us to more generally resolve the need to keep this current (and the reality that if we don't do something to track this that we will not).

Reply:
Created issue: https://github.com/Azure/azure-sdk-for-java/issues/4426